csharp/Program.cs

```diff
@@ -6,7 +6,55 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello C#!");
+            Console.WriteLine("Ingrese numero de tarjeta de credito");
+            string numeroTarjeta = Console.ReadLine();
+            Console.WriteLine(getIssuer(numeroTarjeta));
+
         }
+
+        public static string getIssuer(string tarjeta)
+        {
+            string op1 = tarjeta[0].ToString();
```

Comment:
Please make your variable names more descriptive. By looking at the name, it should give a hint of what kind of value it holds according to the logic implemented